codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
+  - "delivery-service/**/*"  # ignore PoC DS server and lib
```

Comment:
Let's also add the `cli` right away.

Reply:
`cli` doesn't exist yet... but sure I can try to add it and see what happens 😉